.goreleaser.yml

```diff
@@ -0,0 +1,55 @@
+# This is an goreleaser.yaml file which builds mongodb_exporter releases.
+# Make sure to check the documentation at http://goreleaser.com
+# You can run it locally or with your CI, but you should use `make community-release` or `make snapshot`
+# instead of using `goreleaser` directly as some init steps should be done before it runs.
+dist: bin
+builds:
+  - binary: mongodb_exporter
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - netbsd
```

Comment:
… why?

In practice, I think Linux and macOS are enough for us. We _maybe_ could more systems if someone asks, but doing it for is just… I don't… too many files per release. :)

Reply:
Because we had this in master (in promu.yaml) before: https://github.com/percona/mongodb_exporter/blob/master/.promu.yml#L19

Reply:
Let's remove it then. 😄